### PR TITLE
readme hint: running examples

### PR DESCRIPTION
**Pull request by tpltnt**
Hi there,

i added a quick hint on how to run the examples and also linked the guide for details. When people look at the examples folder they want to try things. This way people get up and running fast & they can learn about the details later.

Cheers,
tpltnt

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+You can run any example by executing
+```
+cargo run --example NAME_OF_THE_EXAMPLE
```
Comment (mitchmindtree) [inline examples/README.md]: Would you mind adding the `--release` flag here? E.g.
```
cargo run --release --example NAME_OF_THE_EXAMPLE
```
This way we can improve the chance that the example performs well on their system and leaves a good first impression :)

**Comment by tpltnt** [comment]
I added the release flag

File: examples/README.md (hunk 1 above)
Comment (tpltnt) [reply examples/README.md]: Done

**Comment by mitchmindtree** [comment]
Thanks!

**Pull request merged**